apps/synapse-portal-framework/src/components/PortalSearch/PortalSearchPage.tsx

```diff
@@ -0,0 +1,111 @@
+import { Box } from '@mui/material'
+import { Outlet, useLocation, useNavigate } from 'react-router'
+import { QueryResultBundle } from '@sage-bionetworks/synapse-types'
+import { useCallback, useMemo, useState } from 'react'
+import { SynapseSpinner } from 'synapse-react-client/components/LoadingScreen/LoadingScreen'
+import { PortalSearchTabConfig, PortalSearchTabs } from './PortalSearchTabs'
+import PortalFullTextSearchField from './PortalFullTextSearchField'
+import SearchParamAwareStandaloneQueryWrapper from './SearchParamAwareStandaloneQueryWrapper'
+import {
+  QueryWrapperPlotNavProps,
+  StandaloneQueryWrapperProps,
+} from 'synapse-react-client'
+
+export type PortalSearchPageProps = {
+  selectedTabIndex?: number
+  configs: QueryWrapperPlotNavProps[]
+  searchPageTabs: PortalSearchTabConfig[]
+}
+
+function getQueryCount(queryResultBundleJSON: string) {
+  const queryResultBundle = JSON.parse(
+    queryResultBundleJSON,
+  ) as QueryResultBundle
+  const { queryCount } = queryResultBundle
+  return queryCount
+}
+
+export function PortalSearchPage(props: PortalSearchPageProps) {
+  const { selectedTabIndex, configs, searchPageTabs } = props
+  // Note: Files does not currently enable FTS
+  const [searchPageTabsState, setSearchPageTabsState] = useState<
+    PortalSearchTabConfig[]
+  >([...searchPageTabs])
+  const navigate = useNavigate()
+  const location = useLocation()
+  const onQueryResultBundleChange = useCallback(
+    (
+      tabIndex: number,
+      newQueryResultBundleJSON: string,
+      selectedTabIndex?: number,
```

Comment:
Mostly a straight copy, but I did find something peculiar.  When my onQueryResultBundleChange() was called (whether it was in this useCallback or not), the selectedTabIndex from the parent module was not updated.  I changed it so I pass it in, which solves the issue, but it was unexpected.  The old context/environment was stored and executed when the query was updated (which caused unexpected results)